components/browser/icons/src/test/java/mozilla/components/browser/icons/generator/DefaultIconGeneratorTest.kt

```diff
@@ -42,8 +44,9 @@ class DefaultIconGeneratorTest {
         assertNotEquals(0, generator.pickColor(res, ""))
     }
 
+    @ExperimentalCoroutinesApi
     @Test
-    fun generate() = runBlocking {
+    fun generate() = runTest(UnconfinedTestDispatcher()) {
```

Comment:
Why do we need the `UnconfinedTestDispatcher` here relative to just using `runTest` as everywhere above.

Reply:
Thank you for catching this.
I'd say this was just a quick "replace runBlocking with runTest(UnconfinedTestDispatcher())" from my part.
Went through the code, it seems like we didn't need `runBlocking` and we don't even need `runTest` with any dispatcher here since this test is not using coroutines.
Updated as such.